piptools/scripts/options.py

```diff
@@ -364,3 +370,28 @@ def _get_default_option(option_name: str) -> Any:
     is_flag=True,
     help="Restrict attention to user directory",
 )
+
+build_deps_for = click.option(
+    "--build-deps-for",
+    "build_deps_for_distributions",
+    multiple=True,
+    type=click.Choice(ALL_BUILD_DISTRIBUTIONS),
+    help="Name of a distribution to install build dependencies for; may be used more than once. "
+    "Static dependencies declared in 'pyproject.toml::build-system.requires' will be included as "
+    "well.",
```

Comment:
```suggestion
    help="Name of a distribution to install build dependencies for; may be used more than once. "
    "Static dependencies declared in 'pyproject.toml::build-system.requires' will be included as "
    "well; may be used more than once",
```

Usually, we include "may be used more than once" phrase for `multiple=True` options.

Reply:
Will move the phrase to make it clearly visible.